.circleci/config.yml

```diff
@@ -593,6 +596,40 @@ jobs:
           name: "Go mod tidy"
           command: make mod-tidy && git diff --exit-code
 
+  hive-test:
+    parameters:
+      version:
+        type: string
+        default: develop
+      sim:
+        type: string
+    machine:
+      image: ubuntu-2004:202201-02
+      docker_layer_caching: true
+      resource_class: xlarge
+    steps:
+      - run:
+          command: git clone https://github.com/ethereum-optimism/hive.git .
+      - go/load-cache
+      - go/mod-download
+      - go/save-cache
+      - run: {command: "go build ."}
+      - run:
+          command: |
+            ./hive \
+            -sim=<<parameters.sim>> \
+            -sim.loglevel=5 \
+            -docker.pull=true \
+            -client=go-ethereum,op-geth_optimism-history,op-proposer_<<parameters.version>>,op-batcher_<<parameters.version>>,op-node_<<parameters.version>>,op-contracts_<<parameters.version>> |& tee /tmp/hive.log
```

Comment:
Is there some way to confirm it pulls the correct images for the hive client containers, i.e. if the images that we pushed as dependency are available soon enough on the docker repo to be used in the CI hive build?

Reply:
Hive will fail if the passed-in image doesn't exist on the repo.

Reply:
I confirmed it's pulling the correct containers by cross referencing the params with the logs.